source/module_base/module_fft/fft_base.h

```diff
@@ -0,0 +1,80 @@
+#include <complex>
+#include <string>
+#include "fftw3.h"
+#ifndef FFT_BASE_H
+#define FFT_BASE_H
+template <typename FPTYPE>
+class FFT_BASE
+{
+public:
+
+	FFT_BASE();
+	virtual  ~FFT_BASE(); 
+	
+	// init parameters of fft
```

Comment:
use doxygen style notes

Reply:
Sure, comments have been added to both functions and variables.